Segmentio/Source/Cells/SegmentioCell.swift

```diff
@@ -139,6 +139,8 @@ class SegmentioCell: UICollectionViewCell {
         if style.isWithText() {
             segmentTitleLabel?.textColor = selected ? selectedState.titleTextColor : defaultState.titleTextColor
             segmentTitleLabel?.font = selected ? selectedState.titleFont : defaultState.titleFont
+            backgroundColor = selected ? selectedState.backgroundColor : defaultState.backgroundColor
+            segmentImageView?.tintColor = selected ? selectedState.imageTintColor : defaultState.imageTintColor
```

Comment:
@LaserPatrick In order to also manage style .onlyImage, you can move this code after the if statement.